distribution/pom.xml

```diff
@@ -421,7 +368,7 @@
                            <artifactId>infinispan-server</artifactId>
                            <version>${project.version}</version>
                            <packaging>zip</packaging>
-                           <file>${project.build.directory}/distribution/${infinispan.brand.prefix}-server-${infinispan.brand.version}.zip</file>
+                           <file>${project.build.directory}/distribution/${infinispan.brand.prefix}-${infinispan.brand.version}-server.zip</file>
```

Comment:
Why this renaming ? I prefer infinispan-server-x.y.z instead of infinispan-x.y.z-server

Reply:
Because I was unable to make the assembly and deploy plugins to cooperate. 

If I didn't attach the assembly id, the deploy plugin deploys all the zips with the same name. If I attach the assembly id, the file name changes to have the `-server` at the end of the filename

Reply:
you can tell it to not attach the